sdk/python/jobs/finetuning/standalone/model-as-a-platform/chat-completion/chat_completion_with_model_as_platform.ipynb

```diff
@@ -463,7 +463,7 @@
     "    OnlineRequestSettings,\n",
     ")\n",
     "\n",
-    "online_endpoint_name = f\"{model_name}-\" + short_guid\n",
+    "online_endpoint_name = f\"finetuned-model-\" + short_guid\n",
```

Comment:
Not sure how this appears in model endpoints page. But having Model name in here may be helpful.